Grammar and clarity add-on fixes (#622)
home-assistant/developers.home-assistant: docs/add-ons/configuration.md

| boot | string | yes | `auto` by system and manual or only `manual`.
| ports | dict | no | Network ports to expose from the container. Format is `"container-port/type": host-port`. If host-port is `null` then the mapping is disabled.
| ports_description | dict | no | Network ports description mapping. Format is `"container-port/type": "description of this port"`.

> | host_network | bool | no | If `true`, the add-on run on host network.

Reviewer: ```suggestion
| host_network | bool | no | If `true`, the add-on runs on host network.
```

| hassio_api | bool | no | This add-on can access the Supervisor's REST API. Use `http://supervisor`.
| homeassistant_api | bool | no | This add-on can access to the Home Assistant REST API proxy. Use `http://supervisor/core/api`.
| docker_api | bool | no | Allow read-oly access to docker API for add-on. Works only for not protected add-ons.
| privileged | list | no | Privilege for access to hardware/system. Available access: `NET_ADMIN`, `SYS_ADMIN`, `SYS_RAWIO`, `SYS_TIME`, `SYS_NICE`, `SYS_RESOURCE`, `SYS_PTRACE`, `SYS_MODULE` or `DAC_READ_SEARCH`
| full_access | bool | no | Give full access to hardware like the privileged mode in docker. Works only for not protected add-ons.
| apparmor | bool/string | no | Enable or disable AppArmor support. If it is enable, you can also use custom profiles with the name of the profile.
| map | list | no | List of maps for additional Home Assistant folders. Possible values: `config`, `ssl`, `addons`, `backup` or `share`. Defaults to `ro`, which you can change by adding `:rw` to the end of the name.
| environment | dict | no | A dictionary of environment variable to run add-on.

> | audio | bool | no | Boolean. Mark this add-on to use internal an audio system. We map a working PulseAudio setup into container. If your application does not support PulseAudio, you may need to install: Alpine Linux `alsa-plugins-pulse` or Debian/Ubuntu `libasound2-plugins`.

Reviewer: ```suggestion
| audio | bool | no | Boolean. Mark this add-on to use internal audio system. We map a working PulseAudio setup into container. If your application does not support PulseAudio, you may need to install: Alpine Linux `alsa-plugins-pulse` or Debian/Ubuntu `libasound2-plugins`.
```

| schema | dict | yes | Schema for options value of the add-on. It can be `false` to disable schema validation and use custom options.
| image | string | no | For use with Docker Hub and other container registries.
| timeout | integer | no | Default 10 (seconds). The timeout to wait until the Docker daemon is done or will be killed.
| tmpfs | string | no | Mount a tmpfs filesystem in `/tmpfs`. Valid format for this option is : `size=XXXu,uid=N,rw`. Size is mandatory, valid units (`u`) are `k`, `m`, `g` and `XXX` has to be replaced by a number. `uid=N` (with `N` the uid number) and `rw` are optional.
| discovery | list | no | A list of services that this add-on provides for Home Assistant. Currently supported: `mqtt`
| services | list | no | A list of services that will be provided or consumed with this add-on. Format is `service`:`function` and functions are: `provide` (this add-on can provide this service), `want` (this add-on can use this service) or `need` (this add-on need this service to work correctly).
| auth_api | bool | no | Allow access to Home Assistant user backend.
| ingress | bool | no | Enable the ingress feature for the add-on.

> | ingress_port | integer | no | Default `8099`. For Add-ons that run on the host network, you can use `0` and read the port later via API.

Reviewer: ```suggestion
| ingress_port | integer | no | Default `8099`. For add-ons that run on the host network, you can use `0` and read the port later via API.
```

| ingress_entry | string | no | Modify the URL entry point from `/`.
| panel_icon | string | no | Default: `mdi:puzzle`. MDI icon for the menu panel integration.

> | panel_title | string | no | Default add-on name, but can modify with this option.

Reviewer: ```suggestion
| panel_title | string | no | Default is the add-on name, but can be modified with this option.
```
